api-docs/swagger.yaml

```diff
@@ -0,0 +1,127 @@
+swagger: '2.0'
```

Comment:
@sedroche  I convert the swagger.json file to yaml file as yaml file is a lot easier to read and write. Mind reviewing this?